docs/python_support/python_support.md

```diff
@@ -0,0 +1,748 @@
+# Python servables in OpenVINO Model Server {#ovms_docs_python_support_python_support}
+
+## Introduction
+
+**This feature is in preview, meaning some behviors of the feature as well as user interface are subjects to change in the future versions**
+
+ Starting with version 2023.3, model server supports execution of custom Python code. Users can now create their own servables in Python. Those servables can run simple pre or post processing tasks as well as complex ones like image or text generation. 
+ 
+ Python execution is enabled via [Mediapipe](../mediapipe.md) by built-in `PythonExecutorCalculator` that allows creating graph nodes that execute Python code. That way Python servables can be used as standalone endpoints (single node graphs) or be part of larger Mediapipe solutions.
+
+ Checkout [quickstart guide](quickstart.md) for simple usage example.
+
+ ## Building Docker Image
+
+Publically distributed Docker images support Python, but they do not come with any external modules. If bare Python is all you need then you can use public image directly. Otherwise you need to extend public image with additional layers that will install all the stuff you need for your Python code to work. For example, let's say you need numpy.
+In that case your Dockerfile may look like:
+
+```dockerfile
+FROM openvino/model_server:latest
+USER root
+ENV LD_LIBRARY_PATH=/ovms/lib
+ENV PYTHONPATH=/ovms/lib/python
+RUN apt update && apt install -y python3-pip git
+RUN pip3 install numpy
+ENTRYPOINT [ `/ovms/bin/ovms` ]
+```
+
+TODO: Here we could provide make target with requirements location as a parameter.
+
+### Building Model Server From Source
+In above section you use public Docker image. In case you want to build it from source do the following:
+
+```bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server
+make docker_build MEDIAPIPE_DISABLE=0 PYTHON_DISABLE=0 OV_USE_BINARY=1 RUN_TESTS=0
```

Comment:
It doesn't make sense to me to set varaibles to their default values. It is only distraction.